tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -38,6 +38,11 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Ensure that there is enough free memory for virtual network test
+    my $MEM = $self->get_free_mem();
+    record_info('Detect FREE MEM', $MEM . 'G');
+    die "FREE MEM=$MEM. The SUT needs at least 20G of RAM for virtual network test" if $MEM < 20000;
```

Comment:
Unit seems mismatch between line 43 and line 44 , besides 20G is not equal to  20000 M, but 20*1024M

Reply:
Great, good point. For now, update to use with `test $MEM >= 20` way here.